ods-devenv/scripts/deploy.sh

```diff
@@ -1578,6 +1578,12 @@ function run_smoke_tests() {
     pushd ../ods-quickstarters/tests
         make setup-tests test
     popd
+
+    # clean up after tests
+    for project in $(oc get project | grep "unitt" | awk '{print $1;}')
```

Comment:
Hmm, why not hardcode the exact names? I feel a bit uneasy just grepping for `unitt` ..